script/build/geoip

```diff
@@ -38,11 +39,11 @@ mkdir -p ${pkg_build_dir}
     (
         set -e
         cd libGeoIP
-        make V=0 -j4 install
+        make ${pkg_make_flags} install
```

Comment:
It's generally more usual in bash to use `UPPERCASE` variable names for user-overridable variables. This also ensures there are no conflicts with user-overridable variables and those used internally by the script.

Reply:
I didn't know. I'll see whether it's possible to avoid user overridable variables (maybe it suffices to let `MAKEFLAGS` do the job?), and, if not, I'll make the ones that user can override uppercase.